src/EnergyPlus/EMSManager.cc

```diff
@@ -1695,7 +1650,7 @@ namespace EMSManager {
 
         std::string cNodeName = state.dataLoopNodes->NodeID(NodeNum);
         std::string cComponentTypeName = "System Node Setpoint";
-        std::string cControlTypeName = controlTypeName(SetPointType);
+        std::string cControlTypeName = controlTypeName[static_cast<int>(SetPointType)];
```

Comment:
Should also be `std::string_view`. PS Even if `controlTypeName` were an array of `std::string` and not `std::string_view`, this should be a `const std::string &`, so that you are not doing a copy.